src/node.rs

```diff
@@ -223,10 +223,10 @@ pub struct Element {
     pub name: QualName,
 
     /// The element ID.
-    pub id: Option<LocalName>,
+    pub id: Option<String>,
 
     /// The element classes.
-    pub classes: HashSet<LocalName>,
+    pub classes: HashSet<String>,
```

Comment:
If we include #101, it might be preferable to keep this as `LocalName` as class names should be highly redundant seen over one or even multiple documents. Lazy initialization would then potentially allow avoiding some of the cost of deduplicating these strings when they are never matched for a given element.

Reply:
You are right. Maybe we should consider merging #91 and #101 at the same time after #101 is ready. However, this would require a major update to the crate since #101 causes breaking changes to be made to the API.